git_repo/services/ext/bitbucket.py

```diff
@@ -26,6 +27,9 @@
 import os, json, platform
 
 
+SERVICE_PACKAGE = pybitbucket
```

Comment:
I don't really like that pattern, it feels a bit redundant with the register target decorator. What I'd rather do is either make a new decorator that manages version, or add an argument to `register_target` with the version string.

Reply:
the other better side of that is that you don't need the full loading of the module into the main system, and if a module has a version string that's not standard (i.e. not using `__version__` we're good for that.